cmd/antrea-agent/agent.go

```diff
@@ -294,6 +294,16 @@ func run(o *Options) error {
 		NodePortAddressesIPv6: nodePortAddressesIPv6,
 	}
 
+	if features.DefaultFeatureGate.Enabled(features.SecondaryNetwork) {
+		bridgeConfig := o.config.SecondaryNetwork.OVSBridges[0]
+		ovsSecBridgeClient := ovsconfig.NewOVSBridge(bridgeConfig.BridgeName, ovsDatapathType, ovsdbConnection, ovsconfig.WithRequiredPortExternalIDs(interfacestore.AntreaInterfaceTypeKey))
```

Comment:
Would it cause issues if the user uses a single NIC for both primary and secondary network interface? I remember we used to handles some issues with this scenario.